scripts/linters/pre_commit_linter.py

```diff
@@ -308,6 +316,22 @@ def _get_all_files_in_directory(dir_path, excluded_glob_patterns):
     return files_in_directory
 
 
+def _get_linter_type(linter_type):
+    """This function is used to return the type of linters to be used.
+
+    Args:
+        linter_type: str. Type of linter.
+
+    Returns:
+        list(str). The list of all linter types.
+    """
+    linter_types = ['custom', 'third_party']
```

Comment:
I was thinking make the variables like what we did with `_PARSER`

Reply:
@kevinlee12 I don't get why we need linter_types as global variable. Could you please explain?

Reply:
It's because it affects the entirety of the linter operation. Also, if we were to remove one, it's easily seen and we can do that.

Reply:
Done.